Makefile

```diff
@@ -6,7 +6,7 @@ KMS_URL ?= https://127.0.0.1:8000
 KEYS_DIR ?= ${KMS_WORKSPACE}/sandbox_common
 RUN_BACK ?= true
 CCF_PLATFORM ?= virtual
-JWT_ISSUER_WORKSPACE ?= ${PWD}/jwt_issuer_workspace
+JWT_ISSUER_WORKSPACE ?= ${PWD}/jwt_issuers_workspace/default
```

Comment:
Why?

Reply:
It's point 3 on the "How" section, it just cleans up where the output of the JWT issuer goes because I noticed the makefile overrides the default in the script in a way where the files aren't properly ignored by git